ledger/acctupdates_test.go

```diff
@@ -1175,6 +1177,128 @@ func TestListCreatables(t *testing.T) {
 	listAndCompareComb(t, au, expectedDbImage)
 }
 
+func TestBoxNamesByAppIDs(t *testing.T) {
+	partitiontest.PartitionTest(t)
+
```

Comment:
Add t.Parallel() or explain why it won't work for this test.

Reply:
seems CI would be unhappy about me for not adding `partitiontest.PartitionTest(t)`, or things similar.

Do you know what is expected behavior behind such line of test configuration? If it does not work out for speeding up the test, or other expected behavior, I will adapt your suggestion.

Reply:
Ideally, I want tests to start like this:
```
func TestCreateAndUse(t *testing.T) {
	partitiontest.PartitionTest(t)
	t.Parallel()
```
because it runs that test in parallel with other tests when running locally.  Because of PartitionTest, this doesn't matter much for CI, but it helps locally.

But you can't do that if your test messes around with some global, shared structure.  For example,
```
func TestRekeying(t *testing.T) {
	partitiontest.PartitionTest(t)
	// t.Parallel() NO! This test manipulates []protocol.Consensus
```
is what I do when I have a test that can not be safely run in parallel with other tests.